tests/common/platform/device_utils_dpu.py

```diff
@@ -0,0 +1,123 @@
+"""
+Helper script for DPU  operations
+"""
+
+import time
+import pytest
+from tests.common.utilities import wait_until
+from tests.common.helpers.platform_api import chassis, module
+from tests.platform_tests.api.conftest import platform_api_conn
+from tests.common.devices.sonic import * #noqa
+from tests.common.helpers.assertions import pytest_assert
+
+@pytest.fixture(scope='function', autouse=True)
```

Comment:
Why the fixture is autouse? It's better to not put an autouse fixture in a common util file. Or you can remove the autouse so this fixture can be used more flexibly.

Reply:
And the scope is function, is it necessary to power on the DPUs before each test case.

Reply:
Idea is to run all the cases with having all the dpus turned on.